modules/@apostrophecms/login/index.js

```diff
@@ -642,13 +659,22 @@ module.exports = {
             _id: token.userId
           });
           await self.passportLogin(req, user);
+          // No access to login attempts in the final phase.
```

Comment:
You could solve this by moving the `clearLoginAttempts` call currently made at the start of the requirements phase to here. But I don't think it's critical because we are still logging all the failed username/password attempts which is what matters most to the client.